node/src/store.rs

```diff
@@ -1,14 +1,354 @@
+use std::fmt::Debug;
+use std::sync::atomic::{AtomicU64, Ordering};
+
+use async_trait::async_trait;
+use celestia_types::ExtendedHeader;
+use dashmap::mapref::entry::Entry;
+use dashmap::DashMap;
+use tendermint::Hash;
```

Comment:
```suggestion
use celestia_types::Hash;
```

Reply:
In this case remove also tendermint from `[dependencies]`

Reply:
nice, removed 👍